torch/testing/_internal/mypy_wrapper.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
```

Comment:
It would be useful to add some comments here about what this file is for. Maybe also add in a link to the relevant wiki page?